### DISCUSSION TITLE
Anonymous Product Info for 730?

### DISCUSSION THREAD

**Discussion by thouravi** [ValvePython/steamctl#77] [Q&A]
Is there any possible way or some sort of hack to get this information?
Currently, I get this output:
![image](https://github.com/ValvePython/steamctl/assets/15676184/8f4594e2-0819-42c1-a0eb-49e7d5eb59db)

Also, I would like to know how to keep the user account logged in.
For some reason, it's asking me to enter password and Email Code everytime I use it.

**Comment by woctezuma** [comment]
You probably have to login with an account which owns a license for [CS:GO](https://steamdb.info/app/730/), even a F2P license.
